release-notes/app-catalog/app-platform-announcement.md

```diff
@@ -0,0 +1,51 @@
+# Announcing Giant Swarm App Platform
+We are pleased to announce that the Giant Swarm App Platform is generally available.
+
+In the past few months, you may have noticed that we have been working on this. We have answered questions and fixed issues that we found as a result of the preview. This has helped us shape the platform and bring it to where it is today.
+
+There are two ways to benefit:
+
+* One, install the apps from the Giant Swarm Catalog on your clusters. Some apps we are working towards fully supporting are Kong and Prometheus Operator.
```

Comment:
```suggestion
* One, enjoy peace of mind with managed apps installed from the Giant Swarm Catalog on to your clusters. Some apps we are working towards fully supporting are Kong and Prometheus Operator.
```

Reply:
@puja108 re: "benefits". I made an edit to make it clear it's not installing it is not the benefit. I also want to make sure that it flows well w the rest of the text (so did not get all the text from the website which are clearer on benefits). WDYT?